kura/meta_cluster.py

```diff
@@ -86,9 +86,9 @@ def __init__(
         **kwargs,  # For future use
     ):
         if clustering_model is None:
-            from kura.k_means import KmeansClusteringMethod
+            from kura.cluster import KmeansClusteringModel
 
-            clustering_model = KmeansClusteringMethod(12)
+            clustering_model = KmeansClusteringModel(12)
```

Comment:
what is 12? why is it hardcoded

Reply:
This is older code that will be removed in subsequent PR later in the afternoon